### PR TITLE
test: sequential tests for ember

### PR DESCRIPTION
**Pull request by brendaHuang-2023**
sequential tests for ember

### CODE REVIEW

**Comment by brendaHuang-2023** [comment]
This PR needs the code changes from other test PRs, mainly the added dependencies in test-support/fixtures/ember/addon-template project. Will merge this PR after the others have been merged.

**Comment by brendaHuang-2023** [comment]
THIS PR WILL BE THE LAST OF ALL TEST PRS TO MERGE. Before it's merged, I will need to retrigger the CI tests even though it may have passed earlier.

File: packages/cli/test/commands/migrate/sequential.ember-addon.test.ts
Comment (lynchbomb) [reply packages/cli/test/commands/migrate/sequential.ember-addon.test.ts:31]: ^^^ and the delta from this test vs the other migrate test eg. e2e config, e2e ...

File: packages/cli/test/commands/migrate/sequential.ember-app.test.ts
Comment (lynchbomb) [inline packages/cli/test/commands/migrate/sequential.ember-app.test.ts:99]: these are only checking that `git mv` worked, which isn't super helpful. this test should be focused on running the migration **sequentially**. there should be a delta between running the migration as a single pass (e2e tests) vs in sequence (this test) and then testing against that delta

**Review by lynchbomb — CHANGES_REQUESTED** [review]
lets be sure the tests around sequential migration are focused on just that. eg

- validate the project was partially migrated
- ... run sequential migration on remainder
- validate the entire project is now migrated

//////

- validate the project was partially migrated
- ... run sequential migration on 1 of N child packages
- validate this child package is migrated
- N of N child packages are now migrated

**Pull request closed without merging**
